lib/invisible_ink.rb

```diff
@@ -11,11 +11,11 @@ def read_file
   end
 
   def write_file
-    @count = File.write(@new_file, read_file)
+    File.write(@new_file, read_file)
   end
 
   def print_results
-    print "Created '#{@new_file}' containing #{@count} characters"
+    print "Created '#{@new_file}' containing #{write_file} characters"
```

Comment:
I got rid of the @count instance variable once I realized that I could call the method within the string interpolation.